src/components/NoteDialogComponents/NoteDialogNoteComponents/NoteDialogNoteContent.scss

```diff
@@ -41,14 +41,24 @@
   padding: 0 4px;
   font-size: $text--md;
   line-height: 1.5rem;
-
   &:not(:disabled) {
     &:hover,
     &:focus-visible {
       border-color: var(--accent-color--desaturated-dark);
     }
   }
 }
+.note-dialog__note-content-container {
+  width: 100%;
+  height: 100%;
+}
+
+.note-dialog__marker-edited {
+  color: $color-middle-gray;
+  font-size: $text--base;
+  border: 2px transparent;
```

Comment:
what does the transparent border do?

Reply:
That's old css i used when the marker was displayed in the note header. Forgot to remove it 😅